wca-regulations.md

```diff
@@ -83,6 +83,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2i1) Competitors may use non-electronic aids that do not give an unfair advantage, at the discretion of the WCA Delegate. This includes:
         - 2i1a) Medical/physical aids worn by the competitor (e.g. glasses, wrist brace). As an exception to [Regulation 2i](regulations:regulation:2i), medical aids may be electronic if the competitor does not have comfortable non-electronic alternatives (e.g. if the competitor has a personal hearing aid or pacemaker).
         - 2i1b) Earplugs and earmuffs (but not electronic headphones and earbuds).
+        - 2i1c) Hand warmers.
```

Comment:
Would you mind specifying "non-electronic" here?

Reply:
Yeah, I kind of wanted to be more clear.
The "While competing" of 2i is becoming steadily less sufficient.

How about adding a Guideline instead, to the effect of "2i1c+ [CLARIFICATION] Electronic hand warmers may be used before or after an attempt. Non-electronic hand warmers may be used during an attempt."

Ideally, I'd like to align this with the camera Regs (start of attempt to end of solve, so that interaction is allowed before signing the score card). I don't have time to think through that change right now, though.

Reply:
Added the clarification and made slightly more specific https://github.com/thewca/wca-regulations/pull/701/commits/ce3173a1699c409daba1fbab4d93d37ba6cdce39